samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/AuthorPartListener.java

```diff
@@ -53,7 +53,7 @@ public void processAction(ActionEvent ae) throws AbortProcessingException
     SectionBean sectionBean = (SectionBean) ContextUtil.lookupBean(
                                           "sectionBean");
     // clean it
-    sectionBean.setSectionTitle("");
+    sectionBean.setSectionTitle("Default");
```

Comment:
You need to internationalise the string "Default"

ResourceLoader rc = new ResourceLoader("org.sakaiproject.tool.assessment.bundle.CommonMessages");
sectionBean.setSectionTitle(rc.getString("default"));

Reply:
The string cannot be internationalized, since the code does not distinguish between languages ​​in this case, it only checks that the title string is "default". If you use a different one it won't work.

samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/delivery/SectionContentsBean.java

public String getNonDefaultText()
  {

    if ("Default".equals(text) || "default".equals(text))
    {
      return "";
    }
    return text;
  }